.github/workflows/scan-pull-request.yaml

```diff
@@ -34,11 +34,23 @@ jobs:
           #
           # Valid examples would be
           # - fix: resolve minor issue
-          # - assetindex: update some dependency
-          # - docs(sample5): update docs for configuration
+          # - docs(Sample5): update docs for configuration
           # - feat(management-api)!: change path to access contract agreements
           #
           # Invalid examples would be
           # - Add cool feature
           # - Feature/some cool improvement
-          regex: '(?<type>\w+)(?<scope>(?:\([^()\r\n]*\)|\()?(?<breaking>!)?):(?<subject>.*)?'
+          # - fix: resolve minor issue.
+          regex: '^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)(\(\w+((,|\/|\\)?\s?\w+)+\))?!?: [\S ]{1,49}[^\.]$'
+          allowed_prefixes: 'build,chore,ci,docs,feat,fix,perf,refactor,revert,style,test'
+          prefix_case_sensitive: true
+
+  check-for-assigned-labels:
+    runs-on: ubuntu-latest
+    continue-on-error: false
+    steps:
+      - uses: actions/checkout@v3
+      - uses: agilepathway/label-checker@v1.0.113
+        with:
+          any_of: api,bug,build,dependencies,documentation,enhancement,ids,java,no-changelog,policy,refactoring
```

Comment:
I would propose to choose the labels based on which categories we want to see in the changelog.
e.g. currently now there are only Bugfixes (bug), Documentation (documentation) and New Features & Improvements (*).
I would keep `bug`, `documentation`, `enhancement` (that could be used for `new features`), `no-changelog`. 
maybe also `api`, `dependencies` and `refactoring` (that could be added as categories in the `changelog` task configuration)
but I would avoid the others (`build`, `ids`, `java`, `policy`).

Makes sense?

Reply:
yes.. the problem is that there is no action that allows wildcards, so e.g. changes to the pipeline would need the `build` label as no PRs with 0 labels are allowed. Maybe just skip `ids`, `java`, `policy` as this is too specific? The other ones kind of match the conventional commits prefixes.

Reply:
I would suggest to take a look at adding further categories to the changelog later.. first step is to have labels.